src/main/kotlin/no/nav/medlemskap/regler/funksjoner/AdresseFunksjoner.kt

```diff
@@ -14,8 +14,11 @@ object AdresseFunksjoner {
     fun Adresse.adressensPeriodeOverlapperKontrollPerioden(kontrollPeriode: Periode) =
             Funksjoner.periodefilter(lagInterval(Periode(this.fom, this.tom)), kontrollPeriode)
 
-    fun List<Adresse>.harBrukerNorskAdresseInnenforSiste12Mnd(kontrollPeriode: Periode): Boolean =
-            this.any { it.adresseLandskodeErNorsk() && it.adressensPeriodeOverlapperKontrollPerioden(kontrollPeriode) }
+    fun List<Adresse>.adresserSiste12Mnd(kontrollPeriode: Periode): List<Adresse> =
+            this.filter {it.adressensPeriodeOverlapperKontrollPerioden(kontrollPeriode) }
+
+    fun List<Adresse>.landekodeTilAdresseSiste12Mnd(kontrollPeriode: Periode): List<String> =
```

Comment:
Returnerer en liste, så navngi til landkoder

Reply:
kan den kalles det den heter her da? også i lovalgRegler kalle den for noe sånn: 
val landekoder = landekodeTilAdresseSiste12Mnd...

Reply:
Hvor fant du val landekoder = landekodeTilAdresseSiste12Mnd... i RelgerForLovvalg ?
Her synes jeg i såfall det skal hete val landkoder = landkoderTilAdresserSiste12Mnd

Reply:
Fant det ingen plass, var et forslag til forbedring :)

Reply:
ah, jeg misforstod. Ble bra med de nye endringene :)